### PR TITLE
Adding 6 CMU faculty members affiliated with School of Computer Science and Machine Learning Dept.

### PR DESCRIPTION
**Pull request by ahmaurya**
# Contributing to CSrankings

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

**Comment by fycus-tree** [comment]
Some of these people are not eligible. Artur, for example, is a research professor. Byron is part of ECE and Biomedical, not SCS, etc. 

#305

**Comment by ahmaurya** [comment]
@fycus-tree In that case, I could remove Dubrawski and Gormley as they are not TT. Byron Yu OTOH is TT, is affiliated with SCS, and independently advises doctoral students in Comp. Biology and Machine Learning. I think he satisfies the criteria for inclusion.

**Comment by ahmaurya** [comment]
@fycus-tree Removed Dubrawski and Gormley from the listings.

Btw, could you link to your real name or academic web page on your GH profile page? Fycus Tree doesn't give us much to go by in terms of transparency.

File: csrankings.csv [hunk 1]
```diff
@@ -2046,6 +2046,7 @@ Byron Boots,Georgia Institute of Technology,http://www.cc.gatech.edu/~bboots3,kX
 Byron C. Wallace,Northeastern University,http://www.byronwallace.com,KTzRHmwAAAAJ
 Byron Choi,Hong Kong Baptist University,http://www.comp.hkbu.edu.hk/~bchoi,S6L1eOMAAAAJ
 Byron J. Williams,Mississippi State University,http://www.fsnhp.msstate.edu/docs/staff/Byron%20%20Williams%20CV%20July%202012%20.pdf,t4bGLWUAAAAJ
+Byron Yu,Carnegie Mellon University,http://users.ece.cmu.edu/~byronyu/,Fz3_tukAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:2049]: Not here: https://www.cs.cmu.edu/directory

File: csrankings.csv [hunk 2]
```diff
@@ -4826,6 +4827,7 @@ George Danezis,University College London,http://www0.cs.ucl.ac.uk/staff/G.Danezi
 George Darmiton da Cunha Cavalcanti,UFPE,www.cin.ufpe.br/~gdcc,0dSAy8IAAAAJ
 George E. Nasr,Lebanese American University,http://www.lau.edu.lb/about/governance-policies/executive-officers/george-nasr.php,S9yBpOMAAAAJ
 George F. Pinder,University of Vermont,http://www.uvm.edu/~cems/?Page=employee/profile.php&SM=employee/_employeemenu.html&EmID=18,NOSCHOLARPAGE
+George H. Chen,Carnegie Mellon University,http://www.andrew.cmu.edu/user/georgech/index.html,O8eqJA4AAAAJ
```
Comment (emeryberger) [inline csrankings.csv:4830]: Not here. https://www.cs.cmu.edu/directory

File: csrankings.csv [hunk 3]
```diff
@@ -6579,6 +6581,7 @@ Jeremiah Blocki,Purdue University,https://www.cs.purdue.edu/homes/jblocki,KBvKjb
 Jeremy Aarons,Monash University,http://monash.academia.edu/JeremyAarons,NOSCHOLARPAGE
 Jeremy Blackburn,University of Alabama - Birmingham,https://www.uab.edu/cas/computerscience/people/faculty-directory/jeremy-blackburn,oO1GPO8AAAAJ
 Jeremy Buhler,Washington University in St. Louis,http://www.cse.wustl.edu/~jbuhler,yQXfAWMAAAAJ
+Jeremy C. Weiss,Carnegie Mellon University,http://www.andrew.cmu.edu/user/jweiss2/,BrWvx00AAAAJ
```
Comment (emeryberger) [inline csrankings.csv:6584]: Not listed in the directory for SCS.
https://www.cs.cmu.edu/directory

File: csrankings.csv [hunk 4]
```diff
@@ -15997,6 +16002,7 @@ Yücel Yemez,Koç University,http://network.ku.edu.tr/~yyemez,12csu04AAAAJ
 Z. Cihan Taysi,Yıldız Technical University,https://www.ce.yildiz.edu.tr/personal/cihan?lang=en,NOSCHOLARPAGE
 Z. Meral Özsoyoglu,Case Western Reserve University,http://engineering.case.edu/eecs/faculty-staff,QRrrUU0AAAAJ
 Zac Friggstad,University of Alberta,https://webdocs.cs.ualberta.ca/~zacharyf,NOSCHOLARPAGE
+Zachary Chase Lipton,Carnegie Mellon University,http://zacklipton.com/,MN9Kfg8AAAAJ
```
Comment (emeryberger) [inline csrankings.csv:16005]: Not in the directory.
https://www.cs.cmu.edu/directory

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 1 above)
Comment (ahmaurya) [reply csrankings.csv:2049]: SCS directory is not complete or frequently updated to be current. Different departments maintain their own frequently updated personnel lists. See Byron's affiliation [here](https://www.ml.cmu.edu/people/affiliated-faculty.html) Also see his advisees [here](https://www.ml.cmu.edu/people/phd-students.html)

File: csrankings.csv (hunk 2 above)
Comment (ahmaurya) [reply csrankings.csv:4830]: SCS directory is not complete or frequently updated to be current. Different departments maintain their own frequently updated personnel lists. See George's affiliation [here](https://www.ml.cmu.edu/people/affiliated-faculty.html)

File: csrankings.csv (hunk 3 above)
Comment (ahmaurya) [reply csrankings.csv:6584]: SCS directory is not complete or frequently updated to be current. Different departments maintain their own frequently updated personnel lists. See Jeremy's affiliation [here](https://www.ml.cmu.edu/people/affiliated-faculty.html)

File: csrankings.csv (hunk 4 above)
Comment (ahmaurya) [reply csrankings.csv:16005]: SCS directory is not complete or frequently updated to be current. Different departments maintain their own frequently updated personnel lists. See Lipton's affiliation [here](https://www.ml.cmu.edu/people/affiliated-faculty.html) Also see his two MLD advisees [here](https://www.ml.cmu.edu/people/phd-students.html). He also advises some students in LTI, SCS afaik.

**Comment by fycus-tree** [comment]
@ahmaurya I wasn't planning on it. I do my best to be an open/fair contributor to parts of the GitHub community. If you see a bias, feel free to point it out. But I've no interest in people taking shots at me or my institutions just because they don't like my comments and wish to ascribe malice to them. There's a few clear policies in the [FAQ](http://csrankings.org/faq.html) and I do my best to point them out if I can. 

Emery's ability to maintain this useful website is remarkable. Especially consider the volume of helpful (#564) and not helpful (#771) criticism he receives. He has a thicker skin than me. My own institution would probably be unhappy about some of my comments on here as certain policies may hurt its rankings. 

Other comments on this commit itself
- All of these faculty are only qualified as Affiliated members of the Machine Learning department. All of them have home departments in other Colleges (Heinz, Tepper). ML seems to have a very expansive "affiliated faculty" program. Any affiliated faculty can be the sole advisor for a PhD student in Machine Learning, which is part of the School of Computer Science. 
- These professors tend to recommend prospective PhD students apply to a non-Computer Science PhD program to work with them. 
- Bryon's status is unclear to me; his website lists affiliations, such as Robotics Institute, which that website does corroborate. Also his sole advised PhD students seem to be only in ECE. 
- George seems to also be an ML affiliated faculty. However, his page recommends that you apply to a NOT Computer Science PhD program (the one in Heinz College). 
- Jeremy has the same status. He says **consider applying to Heinz College ... or the University of Pittsburgh PhD Program in Biomedical Informatics**. While he can advise students in ML, he doesn't recommend applying to the ML program (or even CMU alone)
- Leman seems to have the same status. Affiliated with ML, but runs a lab in Heinz and has mostly students from Heinz.
- Zachary seems to have the same status, except as part of the Business school instead of the Heinz College. 
- In general, this machine learning department has 47 [affiliated faculty members](https://www.ml.cmu.edu/people/affiliated-faculty.html) with only 24 [core faculty](https://www.ml.cmu.edu/people/core-faculty.html) not on leave. That's remarkable considering they have less than [100 PhD students](https://www.ml.cmu.edu/people/phd-students.html)
- I'd ask the question, are the faculty in this commit really CS faculty, or are they more like the Informatics faculty at Oxford (#604) or Penn State (#863)

**Comment by ahmaurya** [comment]
@fycus-tree I do not wish to argue the merits (or not) of each inclusion. The only question/criterion that should matter is "Make sure that any faculty you add meet the inclusion criteria. Eligible faculty include only full-time, tenure-track faculty members on a given campus who can solely advise PhD students in Computer Science. Faculty not in a CS department or similar who can advise PhD students in CS can be included regardless of their home department." If the inclusion criterion seems lax to you, you should argue for changing it. But once the criterion is set, the inclusion of faculty who meet the criterion cannot be discretionary or arbitrary.

As you said yourself, "any affiliated faculty can be the sole advisor for a PhD student in Machine Learning, which is part of the School of Computer Science." Thus, the faculty in my commit clearly meet the inclusion criteria. Btw, this is true of all departments in SCS, not just machine learning.

**Comment by ahmaurya** [comment]
@fycus-tree I understand your reasons for staying anonymous. It's just easier, at least for me, to have a discussion with a person than an account. :)

**Comment by fycus-tree** [comment]
@ahmaurya I guess, in the sense that the Machine Learning Department is part of the Computer Science School, they fit the criteria and the letter of the law. However, it could be that the affiliation-rich nature of the department and the FAQ's comment "**the primary audience of CSRankings is prospective graduate students who are seeking a postgraduate degree in Computer Science**", and these faculty don't even recommend students apply to a CS PhD program at CMU means the spirit of the rankings discourages their addition. 

To really nit-pick here, if you apply to Carnegie Mellon University for a [PhD in Computer Science](https://csd.cs.cmu.edu/sites/default/files/CSD-PhD-Handbook-2017-18.pdf), your committee must consist of a Computer Science faculty member (one of about 70 faculty) and 2 faculty from the School of Computer Science (from an approved list). That suggests that only the [Computer Science Department](http://csd.cs.cmu.edu) counts. That is, not Robotics, not Machine Learning, not Computational Biology; none of those appointments allow one to be the sole advisor for a CS PhD student at CMU. ( although your advisor could be... anyone from the college upon specific approval? Unclear to me or my friend who helped me get this info). That would suggest clearing out most of the CMU-affiliated faculty from CS Rankings. So is a PhD in Machine Learning enough to count for CS Rankings (part of the School of Computer Science)? Or is it only for a PhD in CS (Computer Science Department)? 

And if a Machine Learning PhD is enough, then why not Robotics at other schools (#808) or Informatics (#482 #604 #805 #863)?

Edit: Just as a really quick note on personal preferences, I'm a big fan of revising the inclusion criteria to clear this ambiguity, see #480 .

**Comment by fycus-tree** [comment]
@ahmaurya any clarity on why you closed this pull request?

**Pull request closed without merging**